packages/bp/package.json

```diff
@@ -21,7 +21,7 @@
     "@babel/parser": "^7.11.2",
     "@babel/traverse": "^7.11.0",
     "@microsoft/recognizers-text-suite": "^1.3.0",
-    "@socket.io/redis-adapter": "^7.0.1",
+    "@socket.io/redis-adapter": "^7.2.0",
```

Comment:
Using `@socket.io/redis-adapter` >= [v7.1.0](https://github.com/socketio/socket.io-redis-adapter/releases/tag/7.1.0) allow us to re-use the same error handling callback we defined on the Redis client.